src/Handler/Build.php

```diff
@@ -44,7 +44,7 @@ public function __construct($cwd)
         $vagrantTasks = [
             'up --no-provision',
             'provision',
-            'ssh -- "cd /srv/app && make install"',
+            'ssh -- "cd /srv/app && composer install --verbose && bin/console doctrine:database:create --if-not-exists"',
```

Comment:
Looks like something that could be configured per env in a [`manala.yml` file](https://github.com/manala/manala/pull/29#issuecomment-250901383) under a `build.install` key ? 😄